behandlingsprosess/src/main/java/no/nav/ung/sak/domene/behandling/steg/foreslåvedtak/ForeslåVedtakTjeneste.java

```diff
@@ -57,8 +62,14 @@ class ForeslåVedtakTjeneste {
         // TODO: Fiks integrering mot k9-tilbake
 //        aksjonspunktDefinisjoner.addAll(sjekkMotTilbakekrevingTjeneste.sjekkMotÅpenIkkeoverlappendeTilbakekreving(behandling));
 
-        Optional<Aksjonspunkt> vedtakUtenTotrinnskontroll = behandling
-            .getÅpentAksjonspunktMedDefinisjonOptional(AksjonspunktDefinisjon.VEDTAK_UTEN_TOTRINNSKONTROLL);
+        if (BehandlingType.KLAGE.equals(behandling.getType())) {
```

Comment:
burde vi flytte foreslåvedtak for klage i eget steg? Er alt utenfor if'en relevant for klage? 

Alternativt kan jeg omskrive vurderBrev til å også håndtere klage - i den så sjekker jeg om påkrevde brev blir skrevet og hvis det mangler brev for tilfeller så feiler steget.